packages/core/src/filters/Filter.js

```diff
@@ -201,6 +198,63 @@ export class Filter extends Shader
         this.state = new State();
     }
 
+    /**
+     * Input frame required by this filter, as recorded by the last measure
+     * pass.
+     * @returns {PIXI.Rectangle}
+     */
+    get frame()
+    {
+        return this._frame;
+    }
+
+    /**
+     * Whether this filter can be applied without reducing the refresh rate
+     * significantly
+     * @returns {boolean}
+     */
+    get renderable()
+    {
+        return this._renderable;
+    }
+
+    /**
+     * Overridable method called by `measure`. Use this to provide your custom measurements,
+     * by setting `this._frame` and `this._renderable`.
+     * @param {PIXI.Rectangle} targetBounds
+     * @param {PIXI.Rectangle} passBounds
+     * @param {number} padding
+     * @abstract
+     * @see {@link PIXI.Filter#frame}
+     * @see {@link PIXI.Filter#renderable}
+     */
+    onMeasure(targetBounds, passBounds, padding)// eslint-disable-line no-unused-vars
+    {
+        this._frame = passBounds;
+        this._renderable = true;
+    }
+
+    /**
+     * Measures the input frame needed by this filter to calculate pixels
+     * in the pass-output. It should keep the results in `Filter#frame`
+     * @param {PIXI.Rectangle} targetBounds - bounds of the target object
+     * @param {PIXI.Rectangle} passBounds - frame in which output is required
+     * @param {number} padding - padding applied in the target bounds
+     */
+    measure(targetBounds, passBounds, padding)
+    {
+        this.onMeasure(targetBounds, passBounds, padding);
+
+        if (this.frame === null || this.frame === undefined)
+        {
+            throw new Error(`${this.constructor.name}#onMeasure does not set Filter#_frame.`);
```

Comment:
Could you verify that `this.constructor.name` is valid in the minified version? Sometime class names get mangled with minification.

Reply:
Even if it doesn't get mangled here, it might get mangled if pixi is minified in other build environments. 

But since it's for logging an error only so nothing relies on it... might be fine?